Index: templatetags/representations.py
===================================================================
--- templatetags/representations.py	(revision 3)
+++ templatetags/representations.py	(working copy)
@@ -6,8 +6,9 @@
 register = template.Library()
 def get_representation_content(model, representation, context=None):
     opts = model._meta
-    label = "%s.%s" % (opts.app_label, opts.object_name.lower())
-    t = loader.get_template("representations/%s/%s" % (label,representation))
+    template_list = [ "representations/%s/%s/%s" % (opts.app_label, opts.object_name.lower(), representation), 
+                      "representations/%s/%s"    % (opts.app_label, representation), ]
+    t = loader.select_template(template_list)
 
     if context is None:
         context = Context()
@@ -57,7 +58,9 @@
     This tag takes a model and passes it and the context to
     a template in the following location:
     
-    representations/app_label.object_name/[template]
+    representations/app_label/object_name/[template] or
+    representations/app_label/[template] or
+    representations/[template] (whichever comes first)
     {% represent [model] as "[template]" %}
 
     This allows you to have the same representation for the object in numerous
